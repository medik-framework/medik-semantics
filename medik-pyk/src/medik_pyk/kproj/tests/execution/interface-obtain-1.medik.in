[{"tid":1,"id":"ex1","result":"obtainResponse","args":72},{"action":"exit"}]
