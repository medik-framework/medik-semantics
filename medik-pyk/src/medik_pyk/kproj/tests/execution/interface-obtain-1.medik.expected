{"id": "ex1", "tid": 1, "interface": "External", "name": "Obtain", "args": ["hr"]}
72